ardupilot/equipment/power/20008.SmartBatteryContinuous.uavcan

```diff
@@ -0,0 +1,28 @@
+#
+# Smart battery data to be sent continuously (1 - 10Hz)
+#
+
+float16 temperature             # [C]      : Pack mounted thermistor (preferably installed between cells), NAN: field not provided
+float32 current                 # [Ampere] : Postive: defined as a discharge current. Negitive: deifined as a charging current, NAN: field not provided
+float16 voltage                 # [Volt]
+uint32 capacity_consumed        # [mAh]
```

Comment:
yeah I could go either way on it. Figured it might be useful for non-smart batteries (only coulomb counting for example) where they may not actually know the remaining capacity but do know how much has been consumed.

Reply:
Yes. For MAVLink at least we need both. 

To be very precise, a power monitor can really only measure the consumed current. Systems that use this always assume full battery to start with and calculate remaining capacity from consumed current. This assumption is flawed because the power monitor resets on every reboot, and might actually not be full (as you say).

So we need this for power monitors. Probably in MAVLink we'll just highlight this issue with using that kind of hardware. Perhaps recommend only supplying capacity_remaining if that is reliable and consumed current if that is what is reliable.